package.json

```diff
@@ -43,7 +43,7 @@
   "dependencies": {
     "consolidate": "^0.14.0",
     "hash-sum": "^1.0.2",
-    "js-beautify": "^1.6.14",
+    "js-beautify": "1.6.14",
```

Comment:
we should do this once we see that `js-beautify` fails to actually fix it. We should wait a while.

Reply:
Already fixed.  1.7.3 works.  The `js-beautify` project does use semantic versioning so `~1.7.3` would also be reasonable.